consensus/bor/bor.go

```diff
@@ -988,7 +988,11 @@ func (c *Bor) Prepare(chain consensus.ChainHeaderReader, header *types.Header) e
 
 	header.Time = parent.Time + CalcProducerDelay(number, succession, c.config)
 	if header.Time < uint64(time.Now().Unix()) {
-		header.Time = uint64(time.Now().Unix())
+		if c.config.IsTimestampHF(big.NewInt(int64(number))) {
```

Comment:
Not sure why this is required.

Reply:
I thought it's safer since we are making changes in the consensus ?